### PR TITLE
issue-7:Setup Linting

### LINKED ISSUE #7: Setup Linting
@anass412  Please Follow this https://medium.com/@pppped/extend-create-react-app-with-airbnbs-eslint-config-prettier-flow-and-react-testing-library-96627e9a9672

### PR DESCRIPTION
**Pull request by anass412**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,11716 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (Ibrahimrahhal) [inline yarn.lock]: @anass412  let's add this to gitignore & remove it

File: lintstagedrc.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "*.+(js|jsx)": ["eslint --fix", "git add"],
```
Comment (Ibrahimrahhal) [inline lintstagedrc.json]: @anass412 have you tested this out? shouldn't we be using the tsx extension

File: eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        "prettier",
+        "prettier/react"
+    ],
+    "plugins": [
+        "jsx-a11y",
+        "prettier"
+    ],
+    "rules": {
+        "semi": 0,
+        "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
```
Comment (Ibrahimrahhal) [inline eslintrc.json]: @anass412 same here

**Review by Ibrahimrahhal — CHANGES_REQUESTED** [review]
@anass412  Please pull the main branch once you have fixed all the comments

File: yarn.lock (hunk 1 above)
Comment (anass412) [reply yarn.lock]: i will :)

**Review by Ibrahimrahhal — APPROVED** [review]

**Pull request merged**